src/init.cpp

```diff
@@ -1597,12 +1582,7 @@ bool AppInitMain()
     fFeeEstimatesInitialized = true;
 
     // ********************************************************* Step 8: load wallet
-#ifdef ENABLE_WALLET
-    if (!OpenWallets())
-        return false;
-#else
-    LogPrintf("No wallet support compiled in!\n");
```

Comment:
Should this logging be added to the DummyWalletInit Open function?

Reply:
Good idea. This PR seems to have enough ACKs for merging. Let's save this for a follow-up PR?

Reply:
^ This is now up for grabs